src/renderers/shaders/ShaderLib/depth_frag.glsl

```diff
@@ -32,11 +34,11 @@ void main() {
 
 	#if DEPTH_PACKING == 3200
 
-		gl_FragColor = vec4( vec3( 1.0 - gl_FragCoord.z ), opacity );
```

Comment:
Is the `1.0 - ` removal intentional?

Reply:
Wow.  Definitely not.  Thanks for spotting.